### PR TITLE
Edits to PGD PR5651

### PR DESCRIPTION
**Pull request by ebgitelman**

### CODE REVIEW

File: product_docs/docs/pgd/5/reference/catalogs-visible.mdx [hunk 1]
```diff
@@ -857,14 +857,14 @@ is set correctly when the wait relates to PGD.
 
 ### `bdr.stat_relation`
 
-Shows apply statistics for each relation. Contains data only if tracking is enabled via
+Shows apply statistics for each relation. Contains data only if tracking is enabled with
 [`bdr.track_relation_apply`](/pgd/5/reference/pgd-settings.mdx#bdrtrack_relation_apply)
 and if data was replicated for a given relation.
 
-`lock_acquire_time` is only updated if [`bdr.track_apply_lock_timing`](/pgd/5/reference/pgd-settings.mdx#bdrtrack_apply_lock_timing)
+`lock_acquire_time` is updated only if [`bdr.track_apply_lock_timing`](/pgd/5/reference/pgd-settings.mdx#bdrtrack_apply_lock_timing)
```
Comment (ebgitelman) [inline product_docs/docs/pgd/5/reference/catalogs-visible.mdx:864]:
```suggestion
`lock_acquire_time` is updated only if [`bdr.track_apply_lock_timing`](pgd-settings.mdx#bdrtrack_apply_lock_timing)
```

File: product_docs/docs/pgd/5/reference/catalogs-visible.mdx [hunk 2]
```diff
@@ -857,14 +857,14 @@ is set correctly when the wait relates to PGD.
...
 is set to `on` (default: `off`).
 
-The stored relation statistics can be reset by calling
+You can reset the stored relation statistics by calling
 [`bdr.reset_relation_stats()`](/pgd/5/reference/functions-internal.mdx#bdrreset_relation_stats).
```
Comment (ebgitelman) [inline product_docs/docs/pgd/5/reference/catalogs-visible.mdx:868]:
```suggestion
[`bdr.reset_relation_stats()`](functions-internal.mdx#bdrreset_relation_stats).
```

File: product_docs/docs/pgd/5/reference/catalogs-visible.mdx [hunk 3]
```diff
@@ -891,10 +891,10 @@ The stored relation statistics can be reset by calling
 
 ### `bdr.stat_subscription`
 
-Shows apply statistics for each subscription. Contains data only if tracking is enabled via
+Shows apply statistics for each subscription. Contains data only if tracking is enabled with
 [`bdr.track_subscription_apply`](/pgd/5/reference/pgd-settings.mdx#bdrtrack_subscription_apply).
```
Comment (ebgitelman) [inline product_docs/docs/pgd/5/reference/catalogs-visible.mdx:895]:
```suggestion
[`bdr.track_subscription_apply`](pgd-settings.mdx#bdrtrack_subscription_apply).
```

File: product_docs/docs/pgd/5/reference/catalogs-visible.mdx [hunk 4]
```diff
@@ -891,10 +891,10 @@ The stored relation statistics can be reset by calling
...
 
-The stored subscription statistics can be reset by calling
+You can reset the stored subscription statistics by calling
 [`bdr.reset_subscription_stats()`](/pgd/5/reference/functions-internal.mdx#bdrreset_subscription_stats).
```
Comment (ebgitelman) [inline product_docs/docs/pgd/5/reference/catalogs-visible.mdx:898]:
```suggestion
[`bdr.reset_subscription_stats()`](functions-internal.mdx#bdrreset_subscription_stats).
```

File: product_docs/docs/pgd/5/reference/catalogs-visible.mdx [hunk 5]
```diff
@@ -857,15 +857,15 @@ is set correctly when the wait relates to PGD.
 
 ### `bdr.stat_relation`
 
-Shows apply statistics for each relation. Contains data only if tracking is enabled via
+Shows apply statistics for each relation. Contains data only if tracking is enabled with
 [`bdr.track_relation_apply`](/pgd/5/reference/pgd-settings.mdx#bdrtrack_relation_apply)
```
Comment (ebgitelman) [inline product_docs/docs/pgd/5/reference/catalogs-visible.mdx:861]:
```suggestion
[`bdr.track_relation_apply`](pgd-settings.mdx#bdrtrack_relation_apply)
```

File: product_docs/docs/pgd/4/bdr/catalogs.mdx [hunk 6]
```diff
@@ -786,14 +786,14 @@ is set correctly when the wait relates to BDR.
...
+[`bdr.track_relation_apply`](/pgd/4/bdr/configuration.mdx#bdrtrack_relation_apply)
 and if data was replicated for a given relation.
 
-`lock_acquire_time` is only updated if [`bdr.track_apply_lock_timing`](/pgd/4/bdr/configuration.mdx#bdrtrack_apply_lock_timing)
+`lock_acquire_time` is updated only if [`bdr.track_apply_lock_timing`](/pgd/4/bdr/configuration.mdx#bdrtrack_apply_lock_timing)
 is set to `on` (default: `off`).
 
-The stored relation statistics can be reset by calling
+You can reset the stored relation statistics by calling
 [`bdr.reset_relation_stats()`](/pgd/4/bdr/functions.mdx#bdrreset_relation_stats).
```
Comment (ebgitelman) [inline product_docs/docs/pgd/4/bdr/catalogs.mdx:797]:
```suggestion
[`bdr.reset_relation_stats()`](functions.mdx#bdrreset_relation_stats).
```

File: product_docs/docs/pgd/4/bdr/catalogs.mdx [hunk 7]
```diff
@@ -818,10 +818,10 @@ The stored relation statistics can be reset by calling
 
 ### `bdr.stat_subscription`
 
-Shows apply statistics for each subscription. Contains data only if tracking is enabled via
+Shows apply statistics for each subscription. Contains data only if tracking is enabled with
 [`bdr.track_subscription_apply`](/pgd/4/bdr/configuration.mdx#bdrtrack_subscription_apply).
```
Comment (ebgitelman) [inline product_docs/docs/pgd/4/bdr/catalogs.mdx:822]:
```suggestion
[`bdr.track_subscription_apply`](configuration.mdx#bdrtrack_subscription_apply).
```

File: product_docs/docs/pgd/4/bdr/catalogs.mdx [hunk 8]
```diff
@@ -818,10 +818,10 @@ The stored relation statistics can be reset by calling
...
 
-The stored subscription statistics can be reset by calling
+You can reset the stored subscription statistics by calling
 [`bdr.reset_subscription_stats()`](/pgd/4/bdr/functions.mdx#bdrreset_subscripion_stats).
```
Comment (ebgitelman) [inline product_docs/docs/pgd/4/bdr/catalogs.mdx:825]:
```suggestion
[`bdr.reset_subscription_stats()`](functions.mdx#bdrreset_subscripion_stats).
```

File: product_docs/docs/pgd/4/bdr/configuration.mdx [hunk 9]
```diff
@@ -488,13 +488,13 @@ and receivers don't have a writer ID.
     archival, and rotation to prevent disk space exhaustion.
 
 -   `bdr.track_subscription_apply` &mdash; Track apply statistics for
-    each subscription via [`bdr.stat_subscription`](/pgd/4/bdr/catalogs.mdx#bdrstat_subscription).
+    each subscription with [`bdr.stat_subscription`](/pgd/4/bdr/catalogs.mdx#bdrstat_subscription).
```
Comment (ebgitelman) [inline product_docs/docs/pgd/4/bdr/configuration.mdx:491]:
```suggestion
    each subscription with [`bdr.stat_subscription`](catalogs.mdx#bdrstat_subscription).
```

File: product_docs/docs/pgd/4/bdr/configuration.mdx [hunk 10]
```diff
@@ -488,13 +488,13 @@ and receivers don't have a writer ID.
...
     The default value is `on`.
 -   `bdr.track_relation_apply` &mdash; Track apply statistics for each
-    relation via [`bdr.stat_relation`](/pgd/4/bdr/catalogs.mdx#bdrstat_relation).
+    relation with [`bdr.stat_relation`](/pgd/4/bdr/catalogs.mdx#bdrstat_relation).
```
Comment (ebgitelman) [inline product_docs/docs/pgd/4/bdr/configuration.mdx:494]:
```suggestion
    relation with [`bdr.stat_relation`](catalogs.mdx#bdrstat_relation).
```

File: product_docs/docs/pgd/4/bdr/configuration.mdx [hunk 11]
```diff
@@ -488,13 +488,13 @@ and receivers don't have a writer ID.
...
     The default value is `off`.
 -   `bdr.track_apply_lock_timing` &mdash; Track lock timing when tracking
-    statistics for relations via [`bdr.stat_relation`](/pgd/4/bdr/catalogs.mdx#bdrstat_relation).
+    statistics for relations with [`bdr.stat_relation`](/pgd/4/bdr/catalogs.mdx#bdrstat_relation).
```
Comment (ebgitelman) [inline product_docs/docs/pgd/4/bdr/configuration.mdx:497]:
```suggestion
    statistics for relations with [`bdr.stat_relation`](catalogs.mdx#bdrstat_relation).
```

File: product_docs/docs/pgd/4/bdr/catalogs.mdx [hunk 12]
```diff
@@ -786,14 +786,14 @@ is set correctly when the wait relates to BDR.
 
 ### `bdr.stat_relation`
 
-Shows apply statistics for each relation. Contains data only if tracking is enabled via
-[`bdr.track_relation_apply`](/pgd/4/bdr/configuration.mdx#bdrtrack_relation_apply),
+Shows apply statistics for each relation. Contains data only if tracking is enabled with
+[`bdr.track_relation_apply`](/pgd/4/bdr/configuration.mdx#bdrtrack_relation_apply)
```
Comment (ebgitelman) [inline product_docs/docs/pgd/4/bdr/catalogs.mdx:790]:
```suggestion
[`bdr.track_relation_apply`](configuration.mdx#bdrtrack_relation_apply)
```

File: product_docs/docs/pgd/4/bdr/catalogs.mdx [hunk 13]
```diff
@@ -786,14 +786,14 @@ is set correctly when the wait relates to BDR.
...
 and if data was replicated for a given relation.
 
-`lock_acquire_time` is only updated if [`bdr.track_apply_lock_timing`](/pgd/4/bdr/configuration.mdx#bdrtrack_apply_lock_timing)
+`lock_acquire_time` is updated only if [`bdr.track_apply_lock_timing`](/pgd/4/bdr/configuration.mdx#bdrtrack_apply_lock_timing)
```
Comment (ebgitelman) [inline product_docs/docs/pgd/4/bdr/catalogs.mdx:793]:
```suggestion
`lock_acquire_time` is updated only if [`bdr.track_apply_lock_timing`](configuration.mdx#bdrtrack_apply_lock_timing)
```

**Comment by ebgitelman** [comment]
@djw-m LMK if I've either messed up the links (I had tested some in VSC before then editing more of them in here) or if you want me to undo any of these changes.

**Comment by djw-m** [comment]
This is probably the only exception to the relative links rule. If it's reference, wherever it is, it should remain absolute, not relative. It looks odd inside reference, but I'm planning automated tooling to manage and migrate references, so moving them relative could break that.

**Comment by ebgitelman** [comment]
@djw-m I think this is ready for your approval?

**Review by djw-m — APPROVED** [review]
LGTM

**Pull request merged**